.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - Enhancement
+  - Hacktoberfest
+  - Bug
+  - ⚡ To review
+# Label to use when marking an issue as stale
+staleLabel: Stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  🤔 Hace más de 30 días que no hay actividad aquí, si próximamente no hay más se dará
```

Comment:
1. The comment is set as multiline. Maybe this could cause some unexpected behaviour. Furthermore, being the very same sentence, it would make sense to maintain it on the same line 👼
2. We should comment in English since we could have contributions from not Spanish speaking people. Suggestion (I can't make it as a PR change suggestion because I don't know how to do so in a multiline change 😕):
```
markComment: "Issue automatically set as `Stale` after 30 days of inactivity. It will be closed in 7 days if we don't move it forward. We would like to maintain the repo as clean as possible. Thanks for your comprehension 👼"
```

Reply:
With the `>` we can have multiline :)